### PR TITLE
Integrate application-services with shipit

### PR DESCRIPTION
**Pull request by ahal**

### CODE REVIEW

File: frontend/src/views/NewRelease/index.jsx [hunk 1]
```diff
@@ -291,7 +291,7 @@ export default function NewRelease() {
             />
           )}
         />
-        {revision !== '' && (
+        {revision !== '' && selectedBranch.enableTreeherder !== false && (
```
Comment (ahal) [inline frontend/src/views/NewRelease/index.jsx:294]: @gabrielBusta for some reason this isn't working. The `selectedBranch.enableTreeherder` value is `undefined` here, and also `selectedBranch.project` is somehow set to `staging-application-services` even though it should be set to `app-services` in `configs/development.js` :/

Comment (gabrielBusta) [reply frontend/src/views/NewRelease/index.jsx:294]: https://github.com/ahal/shipit/pull/1 allows you to set `enableTreeherder` and `enableReleaseEta` on Git repositories

**Comment by ahal** [comment]
I think this is ready, I was able to successfully run a release on my dev instance. I also just pushed to the `dev` branch if you want to experiment yourselves (deploy currently in progress)

**Comment by ahal** [comment]
Note that successfully creating a release will be blocked on:
https://github.com/mozilla/application-services/pull/5724

Though you can use the `release-v117` branch on `staging-application-services` for testing in the meantime.

**Comment by ahal** [comment]
Nbm, there are test failures to sort out here.

**Review by gabrielBusta — APPROVED** [review]
I gave it a go locally.

[I promoted](https://firefox-ci-tc.services.mozilla.com/tasks/groups/Iox_jFIhTCCeoWE4cKhCOw) and [shipped](https://firefox-ci-tc.services.mozilla.com/tasks/groups/KtX6H6dNTQCAMelLFrU5ew) Application Services.

Things look good. We might run into some issues when we run it in production.

**Comment by gabrielBusta** [comment]
Maybe we can name the product `"application-services"` to match the name of the github projects etc.? That way the release would be named `Application-services-` like the repository. It's fine the way it is tho.

**Comment by ahal** [comment]
I don't really have a strong opinion. Happy to change it, though probably going to leave it as is for now out of laziness :p. I guess we can change it in a follow-up if relman or the app-services team have opinions.

**Pull request merged**